src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+                "id": {
+                    "title": "Actor unique identifier",
+                    "description": "Identifies the Actor with a unique identifier.",
+                    "type": "integer",
+                    "format": "^[0-9]+$"
+                },
+                "title": {
+                    "title": "Title",
+                    "description": "The title given to an actor.",
+                    "type": "string"
+                },
+                "firstNames": {
+                    "title": "First Name",
+                    "description": "The actors first name.",
+                    "type": "string"
+                },
+                "lastName": {
+                    "title": "Last Name",
+                    "description": "The actors last name.",
+                    "type": "string"
+                },
+                "otherNames": {
+                    "title": "Other Names",
+                    "description": "Any other names for the actor.",
+                    "type": "string"
+                },
+                "dateOfBirth": {
+                    "type": "integer",
+                    "title": "Date of Birth",
+                    "maxLength": 13,
+                    "pattern": "^[0-9]{13}$",
+                    "description": "A 13 digit timestamp. Must be over the age of 18."
+                },
+                "address": {
+                    "title": "Address",
+                    "description": "The full address of the Actor.",
+                    "$ref": "#/definitions/Address"
+                },
+                "emailAddress": {
+                    "title": "Email Address",
+                    "description": "The email address of the actor.",
+                    "type": "string"
+                }
+            },
+            "required": [
+                "firstName",
+                "id",
+                "lastName",
+                "address",
+                "dateOfBirth"
+            ],
+            "title": "Actor"
+        },
+        "Address": {
+            "type": "object",
+            "description": "Generic address to be used across other definitions.",
+            "additionalProperties": false,
+            "properties": {
+                "line1": {
+                    "type": "string"
+                },
+                "line2": {
+                    "type": "string"
+                },
+                "line3": {
+                    "type": "string"
+                },
+                "line4": {
+                    "type": "string"
+                },
+                "postCode": {
+                    "type": "string"
+                },
+                "country": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "line1",
+                "postCode",
+                "country"
+            ],
+            "title": "Address"
+        },
+        "Signature": {
```

Comment:
Do we need any additional tracking information here, such as the association with the actor (the equivalent of a relational key back to the actor's "record")? Or the device used to make the signature? Or IP address? Or the user who was logged in when the signature was made? Could someone log in on my behalf, then hand over to me to make the signature while logged into their account (I'm thinking of elderly parents being assisted by their children)? Not sure what our requirements are for validation of a signature in terms of the context where it was made.